src/cli.ts

```diff
@@ -6,6 +6,7 @@ import * as dns from 'dns';
 import * as log from 'loglevel';
 
 import { buildNativefierApp } from './main';
+import { isArgFormatValid } from '../app/src/helpers/helpers';
```

Comment:
Careful with imports automatically-inserted by your editor, they sometimes do weird stuff.

```suggestion
import { isArgFormatValid } from './helpers/helpers';
```